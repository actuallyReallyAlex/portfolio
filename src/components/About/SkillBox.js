import React from 'react'
import PropTypes from 'prop-types'
import { Box, Heading } from 'grommet'

const SkillBox = ({ color, title }) => (
  <Box align="center" direction="row" gap="small" justify="center">
    <Box background={color} height="10px" width="10px" />
    <Heading level="4" margin="none">
      {title}
    </Heading>
  </Box>
)

SkillBox.propTypes = {
  color: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired
}

export default SkillBox
