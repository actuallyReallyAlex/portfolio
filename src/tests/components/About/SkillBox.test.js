import React from 'react'
import SkillBox from '../../../components/About/SkillBox'
import { shallow } from 'enzyme'

const defaultProps = {
  color: 'accent-1',
  title: 'JavaScript'
}

describe('<SkillBox />', () => {
  test('Should render the <SkillBox />.', () => {
    const component = shallow(<SkillBox {...defaultProps} />)
    expect(component).toMatchSnapshot()
  })
})
