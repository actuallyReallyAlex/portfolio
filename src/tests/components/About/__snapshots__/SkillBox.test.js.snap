// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SkillBox /> Should render the <SkillBox />. 1`] = `ShallowWrapper {}`;
